roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -77,6 +77,19 @@ def get_var(task_vars, *keys, **kwargs):
     return value
 
 
+def normalized_release(task_vars):
+    """Helper function to normalize the release format to dotted numbers"""
+    release = str(get_var(task_vars, "openshift_release"))
+    if release.startswith('v'):
+        release = release[1:]  # v3.3 => 3.3
+    return release
+
+
+def normalized_minor_release(task_vars):
+    """Helper function to normalize the release format into x.y minor release"""
+    return '.'.join(normalized_release(task_vars).split('.')[:2])
```

Comment:
Should they look like https://github.com/openshift/openshift-ansible/tree/master/roles/lib_openshift/src/test ? I'm literally just not sure of the mechanics of how to get tests run by CI.